examples/upbound-aws-provider/composite-resources/aurora-postgresql.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: db.awsblueprint.io/v1alpha1
+kind: RelationalDatabase
+metadata:
+  name: test-aurora-postgresql-db
+  namespace: default
+spec:
+  compositionSelector:
+    matchLabels:
+      awsblueprints.io/provider: aws
+      awsblueprints.io/environment: dev
+      cluster.awsblueprints.io/configuration: standard
+  networkConfig:
+    # the below ids needs to be updated before use. 
+    vpcId: vpc-123455
+    subnetIds:
+      - "subnet-123455"
+      - "subnet-123455"
+      - "subnet-123455"
```

Comment:
Can we add these and some of the other parameters in an EnvironmentConfig and grab them from there? If I am a developer, I don't need to know that information.